modules/httpd/httpd_proc.c

```diff
@@ -132,6 +132,24 @@ int httpd_get_val(void *e_data, void *data, void *r_data)
 	return 0;
 }
 
+static char httpd_full_buff[16384];
```

Comment:
Why `16384` ? any special meaning? Maybe this should be a modparam, to be able to increase it? (of course, it will have to be pkg/system allocated at module startup.